<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Soma</class>
 <widget class="QMainWindow" name="Soma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Soma</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_num1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Número 1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_num2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Número 2</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_num1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_num2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_calcular">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Calcular</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
